src/utils/graphql.ts

```diff
@@ -0,0 +1,159 @@
+import { BigNumber } from "ethers";
+import { getWeb3Interface } from "../scripts/web3-interface";
+import { getBlockTimestamp, getDateFromTimestamp } from "./blockchain/helpers";
+import { getGracePeriod, SarcophagusDataSimple } from "./onchain-data";
+import fetch from "node-fetch";
+
+async function queryGraphQl(query: string) {
+  const response = await fetch(process.env.SUBGRAPH_URL!, {
```

Comment:
Probably better to build it into the code, so that when the API is updated all users will have to do is a docker "down-pull-up" to get back up to speed, instead of having to keep track of urls and version numbers.